pulsar-broker/src/main/java/org/apache/pulsar/broker/service/PrecisPublishLimiter.java

```diff
@@ -63,6 +63,13 @@ public boolean isPublishRateExceeded() {
         return false;
     }
 
+    private void tryReleaseConnectionThrottle() {
```

Comment:
please add comments to the code that explain the logic

Reply:
a single sentence might be enough to clarify the logic

Reply:
Done.